Dockerfile

```diff
@@ -76,7 +76,7 @@ RUN pip install \
         check50 \
         Flask \
         Flask-Session \
-        style50==2.2.0 \
+        style50 \
```

Comment:
Is the plan to have `cs50/ecs` do an `--upgrade` on startup?

Reply:
Yes. But we still want it in baseimage, right?

Reply:
si